README.md

```diff
@@ -68,7 +70,7 @@ Please check out the [Contact page](https://github.com/SpecterOps/BloodHound/wik
 ## Licensing
 
 ```
-Copyright 2023 Specter Ops, Inc.
+Copyright 2024 Specter Ops, Inc.
```

Comment:
I believe modifications do not allow us to update the Copyright, only new files can have a newer copyright year

Reply:
Fixed - would you mind approving?